client/lib/generate-password/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+
+import { random, sample } from 'lodash';
```

Comment:
Refactoring `random` and `sample` to not use lodash should be relatively straightforward with `Math.random()`, no? Is this something you're interested to address while you're here?

Reply:
Yes, I think that makes sense. Will push a commit using custom implementations. Long-term I think it'd be a good idea to talk about whether there are lodash functions we want to keep or implement ourselves and keep them at a more central place to mitigate having a few different implementations of the same function – any thoughts?

Reply:
fixed in https://github.com/Automattic/wp-calypso/pull/48762/commits/2c1fc92b1cbd60872d01e22380933f382b6d5e2d

Reply:
Using `Math.random` (or anything based on it) to generate passwords is questionable. `Math.random` uses a pseudo-random generator that is not cryptographically secure. For example, Firefox uses the [xorshift](https://en.wikipedia.org/wiki/Xorshift) algorithm.

We should rewrite this code to use the Web Crypto APIs.

Reply:
> Yes, I think that makes sense. Will push a commit using custom implementations. Long-term I think it'd be a good idea to talk about whether there are lodash functions we want to keep or implement ourselves and keep them at a more central place to mitigate having a few different implementations of the same function – any thoughts?

Yeah, I think we'll soon start talking about that. Removing lodash is one of the things I'd like to see in 2021 :)

> Math.random uses a pseudo-random generator that is not cryptographically secure. 

True. But neither are the lodash functions that were used before.

> We should rewrite this code to use the Web Crypto APIs. 

Sounds like a good plan to me. Not necessarily for this PR, though - it could be fine to add a `@TODO` for now.